src/Forms/ListboxField.php

```diff
@@ -165,6 +166,17 @@ public function getDisabledItems()
         return $this->disabledItems;
     }
 
+    public function getValueForValidation(): mixed
+    {
+        // Unlike other MultiSelectField's, ListboxField allows setting a single value
+        // Wrap single values in an array so that the validation logic can treat it as a multi-select
```

Comment:
```suggestion
        // Unlike other MultiSelectField's, ListboxField allows setting a single value without wrapping it in an array.
        // Ensure values are wrapped in an array here so that the validation logic can treat it as a multi-select
```
This resolves the confusion I had in https://github.com/silverstripe/silverstripe-framework/pull/11449/files#r1851040162

Reply:
Updated